src/extractor/scripting_environment_lua.cpp

```diff
@@ -276,6 +288,71 @@ void Sol2ScriptingEnvironment::InitContext(LuaScriptingContext &context)
         "version",
         &osmium::Way::version);
 
+    struct RelationMemberWrap
+    {
+        explicit RelationMemberWrap(const osmium::RelationMember & member)
+        {
+            init(member);
+        }
+
+        RelationMemberWrap()
+            : item_type(osmium::item_type::undefined)
+        {
+        }
+
+        void init(const osmium::RelationMember & member)
+        {
+            role = member.role();
+            item_type = member.type();
+            id = member.ref();
+        }
+
+        const char* GetRole() const { return role.c_str(); }
+        osmium::item_type GetItemType() const { return item_type; }
+        osmium::object_id_type GetId() const { return id; }
+
+        util::OsmIDTyped ref() const { return util::OsmIDTyped(id, std::uint8_t(item_type)); }
```

Comment:
conversion of osmium::item_type to `std::uint8_t` must be a hidden in `OsmIDTyped`

Reply:
You talk about store `std::uint8_t` instead of `osmium::item_type` or something else?